### PR TITLE
Add -uj flag that allows to remove Jira issue linking from the current branch

### LINKED ISSUE #9: Remove Jira issue from branch

### PR DESCRIPTION
**Pull request by macanton**
## Summary
Current PR adds '-uj XXXX' flag that allows to remove specified Jira issue code from the branch name

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -158,6 +160,26 @@ func linkJiraIssueToCurrentBranch() {
...
+	if err != nil {
+		printErrorToConsole(err)
+		return
+	}
+
+	issueKey := config.GetIssueKey()
+	updatedBranchName := git.RemoveIssueKeysFromBranchName([]string{issueKey}, currentBranchName)
+
+	output, err := git.UpdateCurrentBranchName(updatedBranchName)
+	if err != nil {
```
Comment (Den1k22) [inline main.go:174]: What do you think to move error check to utils or something like that?

File: pkg/git/utils.go [hunk 2]
```diff
@@ -33,6 +33,16 @@ func PrependIssueKeysToBranchName(issueKeys []string, branchName string) (string
 	return strings.Join(branchNameSubstrings, config.Options.IssueBranchSeparator), nil
 }
 
+// RemoveIssueKeysFromBranchName remove issue keys from branch name
+func RemoveIssueKeysFromBranchName(issueKeys []string, branchName string) string {
+	substrings := strings.Split(branchName, config.Options.IssueBranchSeparator)
```
Comment (Den1k22) [inline pkg/git/utils.go:38]: Usually, "sub" is written with "-". So what do you think to rename `substrings` to `subStrings`

**Review by Den1k22 — CHANGES_REQUESTED** [review]

File: main.go (hunk 1 above)
Comment (macanton) [reply main.go:174]: I thought about it, but as we do not have any additional functionality like logging errors to external service or similar and in some cases we may return immediately and in other continue - I didn't find any good abstraction that solves current 'code duplication' :)

So for now it is enough that we just extract printing to console which can be further used for functionality extension :)

File: pkg/git/utils.go (hunk 2 above)
Comment (macanton) [reply pkg/git/utils.go:38]: Good point, but when I was thinking about it I found that 'substring' in most of the cases is written without '-' indication  in software
https://www.w3schools.com/java/java_ref_string.asp

Comment (Den1k22) [reply pkg/git/utils.go:38]: True, also for go lang https://www.dotnetperls.com/substring-go

**Review by Den1k22 — APPROVED** [review]

**Pull request merged**